src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,29 @@
+package racingcar.domain;
+
+import racingcar.dto.RacingCarStatusResponse;
+
+public class Car {
+    private final Name name;
+    private final Position position;
+
+    public Car(String name, int position) {
+        this.name = new Name(name);
+        this.position = new Position(position);
+    }
```

Comment:
```suggestion
    public Car(String name) {
        this(name, 0);
    }
    
    public Car(String name, int position) {
        this.name = new Name(name);
        this.position = new Position(position);
    }
```

생성자 체이닝을 통해서 자동차 생성의 편의를 도와줄 수도 있겠네요.